fitting/dijetfit.py

```diff
@@ -541,6 +541,11 @@ def dijetfit(options):
 
 def fitting_options():
     parser = optparse.OptionParser()
+    parser.add_option("--scale_j_unc", type=float, default=0.01,
+                      help="Uncertainty on signal mean from JES")
+    parser.add_option("--mjj_max", type=float, default=0.035,
+                      help="Uncertainty on signal width from JER")
```

Comment:
This conflicts with the similarly named option below

Reply:
Oops, fixed now